ext-src/extension.ts

```diff
@@ -94,12 +139,28 @@ function errorToast(msg: string) {
 }
 
 export function activate(context: vscode.ExtensionContext) {
-  // @ts-ignore
-  userSession['eth-config'] = {};
-  // @ts-ignore
-  userSession['user-session-config'] = {
-    'compile': {},
+  let userSession: object = {
```

Comment:
We talked about loading user configuration from `package.json` `configuration`. Why settings are here?